src/ParMetis-4.0.3/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 2.8...3.5)
```

Comment:
This is also in #177, but I needed it here for testing. If that's merged first, we can rebase and this will be gone.